pyanaconda/core/kernel.py

```diff
@@ -162,5 +169,18 @@ def items(self):
         """
         return self._data.items()
 
+    def items_raw(self):
+        """Return an iterator over all arguments in their raw form (with prefixes).
+
+        TODO: DO NOT USE THIS METHOD! This workaround will be removed
+              when 'inst.' prefix is not supported.
+        """
+        for key, val in self._data.items():
+            if key in self._prefixes:
+                yield ("{}{}".format(BOOT_ARG_PREFIX, key), val)
+                continue
```

Comment:
Without the `continue` statement it will yield `inst.stage2` and then `stage2` because it won't take next item from the loop but return on the same place and take the value from 2 lines below.